hack/make-rules/vet.sh

```diff
@@ -23,9 +23,16 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 cd "${KUBE_ROOT}"
 
+# If called directly, exit.
+if [ "${CALLED_FROM_MAIN_MAKEFILE:-""}" == "" ]; then
```

Comment:
I verified it worked without, but cleaned it up anyways to stick with convention.